tests/serialization/test_dag_serialization.py

```diff
@@ -856,7 +860,7 @@ def test_no_new_fields_added_to_base_operator(self):
             'executor_config': {},
             'inlets': [],
             'label': '10',
-            'max_retry_delay': None,
+            'max_retry_delay': timedelta(0, 600),
```

Comment:
This should still be None as we haven't change the default value

Reply:
Gotcha 👍